app/templates/cli.js

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env node
+'use strict';
+var chalk = require('chalk');
+var meow = require('meow');
+var <%= camelModuleName %> = require('./');
+
+var cli = meow({
+	help: [
+		'Usage',
+		'  $ <%= moduleName %> [input]',
+		'',
+		'Options',
```

Comment:
I think the current order is best. Most of the the you are looking for a usage example first and foremost.